state/store_test.go

```diff
@@ -329,24 +333,16 @@ func TestSaveRetainHeight(t *testing.T) {
 }
 
 func TestMinRetainHeight(t *testing.T) {
-	stateDB := dbm.NewMemDB()
-	stateStore := sm.NewStore(stateDB, sm.StoreOptions{
-		DiscardABCIResponses: false,
-	})
-	require.NoError(t, initStateStoreRetainHeights(stateStore, 0, 0, 0))
-	pruner := sm.NewPruner(
-		stateStore,
-		nil,
-		log.TestingLogger(),
-		sm.WithPrunerCompanionEnabled(),
-	)
```

Comment:
Why are you deleting these lines? They are not related to the changes in your PR and should probably be left as they are.

Reply:
The reason is that,by deleting this you have accidentally deleted the `initStatStoreRetainHeights` part which is probably why you had to replace the condition below to the height being 10 instead of 0